src/webpage/direct.ts

```diff
@@ -153,8 +153,9 @@ class Group extends Channel{
 		const div = document.createElement("div");
 		Group.contextmenu.bindContextmenu(div, this,undefined);
 		this.html = new WeakRef(div);
-		div.classList.add("channeleffects");
+		div.classList.add("flexltr","memberinfo");
```

Comment:
why memberinfo?

Reply:
new class I made for this and the members list since it should look the same!

Reply:
maybe it should be renamed then?

Reply:
sure! what should we call it

Reply:
What do you think it is? You're the one who made it lol

Reply:
Well it was named memberinfo because it had information about a member (icon/name/status and activity in the future) but I agree it's a weird name. Need to brainstorm a bit, maybe I'll ask some friends what they'd call it?

Reply:
I'm just not sure what it's meant to be, liststyle maybe?

Reply:
Oh that sounds good, will change